test/test_digest_utils.rb

```diff
@@ -66,17 +66,17 @@ def test_integrity_uri
     sha256 = Digest::SHA256.new.update("alert(1)")
     sha512 = Digest::SHA512.new.update("alert(1)")
 
-    assert_equal "ni:///sha-256;bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF-pI",
+    assert_equal "sha-256-bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF+pI=",
       integrity_uri(sha256)
```

Comment:
is `"sha-256-bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF+pI="` without a `type:` okay?